File: python/1132.py
x = int(input())
y = int(input())

if x > y:
    w = x
    z = y
else:
    w = y
    z = x

sum = 0

for i in range(z,w+1):
    if i % 13 == 0:
        continue
    else:
        sum += i

print(sum)

File: lua/1132.lua
local x = tonumber(io.read())
local y = tonumber(io.read())

local sum = 0
local smlst
local bgst

if x > y then
    smlst = y
    bgst = x
else
    smlst = x
    bgst=y
end

for i=smlst, bgst, 1 do
    if i % 13 ~= 0 then
        sum = sum + i
    end
end

print(sum)